### DISCUSSION TITLE
Modals Manager - Confirm Modal Delete Account

### DISCUSSION THREAD

**Discussion by mhanifmuhsin** [mantinedev/mantine#3408] [Q&A]
I have a problem implementing the modal manager, the modal delete account doesn't run smoothly when it closes, here is a sample video that happened the same as in the documentation :
[https://www.loom.com/share/b10208b9eab24e98aaa7551d4966d6e9](url)
what should i fix in the code ?
thank's

**Comment by Sajarin-M** [comment]
Issue resolved in v6. check #3409.
